docker/gcp_pudl_etl.sh

```diff
@@ -125,7 +124,7 @@ function zenodo_data_release() {
 function notify_slack() {
     # Notify pudl-deployment slack channel of deployment status
     echo "Notifying Slack about deployment status"
-    message='# `${BUILD_ID}` status\n\n'
+    message="# `${BUILD_ID}` status\n\n"
```

Comment:
Ugh duh. I had been fiddling with the single/double quotes at some point and left this in the wrong state. Thank you.

Reply:
How many different kinds of quotes can there be omfg.